web/vtadmin/src/components/routes/workflows/WorkflowActions.tsx

```diff
@@ -44,11 +48,31 @@ interface CancelWorkflowOptions {
     keepRoutingRoules: boolean;
 }
 
+interface SwitchTrafficOptions {
+    enableReverseReplication: boolean;
+    force: boolean;
+    initializeTargetSequences: boolean;
+    maxReplicationLagAllowed: number;
+    timeout: number;
+    tabletTypes: number[];
+}
+
+const DefaultSwitchTrafficOptions: SwitchTrafficOptions = {
+    enableReverseReplication: true,
+    force: false,
+    initializeTargetSequences: false,
+    maxReplicationLagAllowed: 30,
+    timeout: 30,
+    tabletTypes: [1, 2, 3],
```

Comment:
It would be great if we can use `topodata.TabletType`s here instead of plain numbers 😄

Reply:
+1 to that! Sorry, I meant to come back and comment on this but then forgot. We should use the existing consts:
```
topodatapb.TabletType_PRIMARY, topodatapb.TabletType_REPLICA, topodatapb.TabletType_RDONLY
```

Reply:
done.